templates/base.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <title>Taxi Service</title>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" integrity="sha384-zCbKRCUGaJDkqS1kPbPd7TveP5iyJE0EjAuZQTgFLD2ylzuqKfdKlfG/eSrtxUkn" crossorigin="anonymous">
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-sm-2">
+
+                {% block sidebar %}
+                {% endblock %}
+
+            </div>
+            <div class="col-sm-10">
+                <h1>Taxi Service Home</h1>
+                <p>You are welcome!</p>
+
+                <h4>Taxi Service Content:</h4>
```

Comment:
this should be in `block content` because this text is not supposed to be on each page